### ISSUE TITLE
Problems with instructions

### ISSUE THREAD

**Issue by mattitepposeppo** [MicrosoftDocs/visualstudio-docs#3909]
I'm totally new to this and I'm not sure if I'm following the instructions correctly or are they faulty. I had really hard time to get the project build properly.

First of all the instructions doesn't say you need @vue/cli package. I had to install it manually using command prompt.

After that the project didn't build but instead gave me weird error. I tried to build it from command prompt and got error message that I need @vue/cli-service-global package. I installed that from command prompt.

Then I got really weird error: 'vue-cli-service' is not recognized as an internal or external command, operable program or batch file.

I had to google it up and found that there should be vue-cli-service.cmd file in \node_modules\.bin folder. I didn't have .bin folder there. I found out that I need to run npm install from command prompt to get it working. I took ages.

After that I managed to build the project.

Last, a little wish. Now when I got working project, I can't find any instruction how to use the compiled component in my ASP.NET Core project. Like, do I have to reference it (no, it broke the build). How do I use the component in .cshtml file. How do I reference the component in my JavaScript. And so on.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 59adf9c7-5894-84ba-32ee-df842aa4658d
* Version Independent ID: 50683e7c-4b31-1773-f20e-00732a0ffbcc
* Content: [Quickstart: Create your first Vue.js app - Visual Studio](https://docs.microsoft.com/en-us/visualstudio/javascript/quickstart-vuejs-with-nodejs?view=vs-2019#feedback)
* Content Source: [docs/javascript/quickstart-vuejs-with-nodejs.md](https://github.com/MicrosoftDocs/visualstudio-docs/blob/master/docs/javascript/quickstart-vuejs-with-nodejs.md)
* Product: **visual-studio-windows**
* Technology: **vs-javascript**
* GitHub Login: @Mikejo5000
* Microsoft Alias: **mikejo**

**Comment by WilliamAntonRohm** [comment]
@mattitepposeppo -- thank you for your feedback; we are looking into it.

In addition, here are a few options where you might consider asking your question: 
-	[https://social.msdn.microsoft.com/Forums/vstudio](https://social.msdn.microsoft.com/Forums/vstudio)
-	[https://www.stackoverflow.com](https://www.stackoverflow.com)
-	[https://www.reddit.com/r/VisualStudio/](https://www.reddit.com/r/VisualStudio)

As well, you might also consider using the [Report a Problem](https://docs.microsoft.com/visualstudio/ide/how-to-report-a-problem-with-visual-studio) tool to report the issue.

**Comment by Mikejo5000** [comment]
I'll plan to recheck these instructions.

**Comment by Mikejo5000** [comment]
I made a few updates to the doc here to match the current templates. Overall, no repro on 16.4--I think the issue has been fixed in a VS update (updated project template). I do see that you get an error in the current TypeScript template if you try to Clean the Solution (per instructions), likely because the template doesn't use the same config, specifically <PostBuildEvent>npm run build</PostBuildEvent>. I updated these instructions.
